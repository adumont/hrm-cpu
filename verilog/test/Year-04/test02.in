43 f1
10 ee
18 7a
38 87
22 4f
34 f8
02 a8
38 f0
42 22
53 3a
56 a2
07 29
18 50
11 d4
55 88
49 66
27 89
16 fd
25 cd
42 cc
