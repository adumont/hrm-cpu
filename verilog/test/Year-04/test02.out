ee
f1
87
7a
f8
4f
f0
a8
3a
22
29
a2
d4
50
66
88
fd
89
cc
cd
